SitumWayfinding/Classes/Positioning/PositioningViewController.swift

```diff
@@ -1308,4 +1318,66 @@ extension PositioningViewController {
     }
 }
 
-
+extension PositioningViewController {
+    func initializeChangeOfFloorMarker() {
+        let icon = self.scaledImage(
+            image: UIImage(named: "change_floor")!,
+            scaledToSize: CGSize(width: 40.0, height: 40.0)
+        )
+        let markerView = UIImageView(image: icon)
+        self.changeOfFloorMarker.iconView = markerView
+        self.changeOfFloorMarker.position = CLLocationCoordinate2D(latitude: 0, longitude: 0)
+        resetChangeOfFloorMarker()
+    }
+    
+    func showChangeOfFloorMarker(position: CLLocationCoordinate2D) {
+        self.changeOfFloorMarker.position = position
+        self.changeOfFloorMarker.map = self.mapView
+    }
+    
+    func scaledImage(image:UIImage, scaledToSize newSize:CGSize) -> UIImage{
+        UIGraphicsBeginImageContextWithOptions(newSize, false, 0.0)
+        image.draw(in: CGRect(x: 0, y: 0, width: newSize.width, height: newSize.height))
+        let newImage:UIImage = UIGraphicsGetImageFromCurrentImageContext()!
+        UIGraphicsEndImageContext()
+        return newImage
+    }
+    
+    func updateChangeOfFloorMarkerForSelectedFloor(segment: SITRouteSegment, selectedFloor: SITFloor) {
```

Comment:
Estas repetindo o selectedFloor no nome do método e nos atributos, de https://www.swift.org/documentation/api-design-guidelines/#conventions, revisa a parte de "Otherwise, if the first argument forms part of a grammatical phrase, omit its label,"

Reply:
updateChangeOfFloorMarkerForSelectedFloor(SITFloor, withFloorSegment:SITRouteSegment)